datahub-web-react/src/app/homeV3/template/components/addModuleMenu/useAddModuleMenu.tsx

```diff
@@ -40,8 +40,11 @@ export default function useAddModuleMenu(
     modulesAvailableToAdd: ModulesAvailableToAdd,
     position: ModulePositionInput,
     closeMenu: () => void,
-): MenuProps {
-    const { addModule } = usePageTemplateContext();
+) {
+    const {
+        addModule,
+        createModuleModalState: { open: openModal },
```

Comment:
somehow we're getting lots of issues with this line according to github actions and the tests that are running: 
<img width="942" height="809" alt="image" src="https://github.com/user-attachments/assets/eac50643-731e-44a4-9ca9-b0f3172a10b9" />